inc/vcf/summary_report_writer.hpp

```diff
@@ -24,88 +24,45 @@ namespace ebi
 {
   namespace vcf
   {
+
     /**
-     * Class that tells whether an error should be written or skipped.
+     * Stores the count and first line of appearance of an error message
+     */
+    struct ErrorSummary
+    {
+      size_t appearances;
```

Comment:
I think the most appropriate word here would be `occurrences` instead of `appearances`?